tarofreighter/chain_porter.go

```diff
@@ -583,6 +583,14 @@ func adjustFundedPsbt(pkt *tarogarden.FundedPsbt, anchorInputValue int64) {
 // stateStep attempts to step through the state machine to complete a Taro
 // transfer.
 func (p *ChainPorter) stateStep(currentPkg sendPackage) (*sendPackage, error) {
+	// Notify subscribers that the state machine is about to execute a
+	// state.
+	stateEvent := ExecuteSendStateEvent{
```

Comment:
Same here re using the pointer (`&ExecuteSendStateEvent{`) on struct initialization instead of when using it. Just to be consistent throughout the code base.